pavelib/utils/test/suites/bokchoy_suite.py

```diff
@@ -128,6 +128,7 @@ def cmd(self):
             "--with-xunit",
             "--xunit-file={}".format(self.xunit_report),
             "--verbosity={}".format(self.verbosity),
+            "--pdb" if self.pdb else "",
```

Comment:
For this one, it could be added to `extra_args` or appended to the `cmd` if it is set.